core.api/src/main/java/ru/tinkoff/qa/neptune/core/api/properties/GeneralPropertyInitializer.java

```diff
@@ -86,10 +93,15 @@ public static synchronized void refreshProperties(File file) {
      * and instantiates system properties.
      */
     public synchronized static void refreshProperties() {
```

Comment:
Look at other methods' signature. They started like "static synchronized", So apply this towards this method. 
public static synchronized void refreshProperties()